packages/react-core/src/demos/DataList/examples/DataListActions.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import {
+  Button,
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  DataListAction,
+  Dropdown,
+  DropdownList,
+  DropdownItem,
+  MenuToggle,
+  MenuToggleElement,
+  PageSection,
+  PageSectionVariants,
+  TextContent,
+  Text
+} from '@patternfly/react-core';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import { DashboardWrapper } from '@patternfly/react-core/dist/js/demos/DashboardWrapper';
+
+export const DataListActions: React.FunctionComponent = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [isDeleted, setIsDeleted] = React.useState(false);
+
+  const onToggle = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const onSelect = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <React.Fragment>
+      <DashboardWrapper mainContainerId="main-content-datalist-view-actions" breadcrumb={null}>
+        <PageSection variant={PageSectionVariants.light}>
+          <TextContent>
+            <Text component="h1">Projects</Text>
+            <Text component="p">This is a demo that showcases PatternFly Data List</Text>
+          </TextContent>
+        </PageSection>
+        <DataList aria-label="single action data list example ">
+          {!isDeleted && (
+            <DataListItem aria-labelledby="single-action-item1">
+              <DataListItemRow>
+                <DataListItemCells
+                  dataListCells={[
+                    <DataListCell key="primary content">
+                      <span id="single-action-item1">Single actionable Primary content</span>
+                    </DataListCell>,
+                    <DataListCell key="secondary content">Single actionable Secondary content</DataListCell>
+                  ]}
+                />
+                <DataListAction
+                  aria-labelledby="single-action-item1 single-action-action1"
+                  id="single-action-action1"
+                  aria-label="Actions"
```

Comment:
These 3 props can be removed from DataListAction here and the other instance of the component. They're not currently being applied anywhere internally.

Ideally they could be applied to the Button (or whatever interactive action) that is passed as children. Depending on whether this demo should be more 1:1 to the Core demo (where each DataListItem as the same "Delete" and "Link" action visually), then we could move the aria-labelledby and id props to those components.

Reply:
> These 3 props can be removed from DataListAction here and the other instance of the component. They're not currently being applied anywhere internally.
> 
These 3 props are mandatory fields , so we can't remove
> Ideally they could be applied to the Button (or whatever interactive action) that is passed as children. Depending on whether this demo should be more 1:1 to the Core demo (where each DataListItem as the same "Delete" and "Link" action visually), then we could move the aria-labelledby and id props to those components.

Reply:
@tlabaj would there be any objection to either marking these props as optional or removing them from the DataListAction interface? We'd still be destructuring them where are are currently so they wouldn't actually be applied to anything, so it shouldn't really cause any issues. It would just prevent us from having to pass in props that aren't doing anything like this.

Reply:
We do have an issue tracking this https://github.com/patternfly/patternfly-react/issues/9823 it just needs to be prioritized.